packages/react-components/src/components/FileDownloadCards.tsx

```diff
@@ -155,7 +169,7 @@ export const _FileDownloadCards = (props: _FileDownloadCards): JSX.Element => {
     [props]
   );
 
-  if (!fileMetadata || fileMetadata.length === 0) {
+  if (!fileMetadata || fileMetadata.length === 0 || fileMetadata[0].attachmentType !== 'fileSharing') {
```

Comment:
I think its safe to assume that if the first item in the `fileMetadata` is not a `fileSharing` type we don't want to display the `FileDownloadCard`. Since you can't have both `fileSharing` and `teamsInlineImage` in the same list / message.

Reply:
I don't think this is correct. fileMetadata is for used only for acs chat. For interop, attachmentType won't be in fileMetadata property

Reply:
Would be nice to add comment here to explain what's going on, I usually can't remember what I thought when I look at code I wrote one month ago